FrameworkSystem/Client/MonitoringClient.py

```diff
@@ -68,12 +63,18 @@ def __init__(self):
     time.sleep(0.1)
 
   def flush(self, allData=False):
-    """ This flush method calls the flush method from the gMonitor object.
+    """ This method is used to periodically flush the data
+        :type allData: bool
```

Comment:
You need a line between the space and the params (true everywhere, I won't repeat it)